webpack.config.js

```diff
@@ -0,0 +1,16 @@
+const defaults = require("@wordpress/scripts/config/webpack.config");
+const path = require("path");
+
+module.exports = {
+	...defaults,
+	entry: {
```

Comment:
I've ejected into a custom wepback.config.js so that each script can be enqueued seperately. 

This allows us to export functions in the `index.js` and enqueue them to be available in plugins and the app. 

Since functions like `registerPlugin` and the redux store(s) need to be available _before_ plugins and the App call those functions, and we're not importing them via an NPM build script, we need to make sure the enqueue order has the index.js script enqueued first (as a dependency of anything that needs those functions).